CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## Unreleased
+
+### New features
+
+#### Add custom classes to the count message in the character count component
+
+You can now add custom classes to the count message in the character count component, which has been changed to use the hint component rather than custom markup.
```

Comment:
When writing CHANGELOG notes we try to focus on what the user needs to know, so how we fixed or updating something is not required to understand the new feature.

So this could instead focus on how to use the new `countMessageClasses` option rather than the internal details on how we implemented this change.